### PR TITLE
Lesson 4 task 8 - Person class 

### PR DESCRIPTION
**Pull request by jean-farinea**
1. Person class created with equals and hashcode overide methods

### CODE REVIEW

File: src/main/java/lesson4/task8/Person.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    public Person(String name, String surname, String email) {
+        this.name = name;
+        this.surname = surname;
+        this.email = email;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        return ((Person) obj).email.equals(this.email);
```
Comment (strzeleckik) [inline src/main/java/lesson4/task8/Person.java]: so the person is equal to object of the same type when emails are equal? :)

Comment (strzeleckik) [reply src/main/java/lesson4/task8/Person.java]: Maybe it would be better to add other fields to this comparison?

Comment (jean-farinea) [reply src/main/java/lesson4/task8/Person.java]: Adjusted, please review.

**Review by strzeleckik — APPROVED** [review]
good!

**Pull request merged**